### PR TITLE
[msvc] Fix static linking gflags and glog

### PR DESCRIPTION
**Pull request by KindDragon**
gflags and glog require some compile definitions when linking statically
/cc @Orvid

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -152,15 +179,12 @@ set(FOLLY_LINK_LIBRARIES
 
 target_include_directories(folly_base
   PUBLIC
-    ${DOUBLE_CONVERSION_INCLUDE_DIR}
-    ${LIBGFLAGS_INCLUDE_DIR}
-    ${LIBGLOG_INCLUDE_DIR}
-    ${LIBEVENT_INCLUDE_DIR}
-    $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}>
```
Comment (Orvid) [inline CMakeLists.txt]: This line is still needed, for folly-config.h.

Comment (KindDragon) [reply CMakeLists.txt]: Ok

**Comment by KindDragon** [comment]
The last issue for me

File: CMakeLists.txt (hunk 1 above)
Comment (KindDragon) [reply CMakeLists.txt]: I returned it

**Pull request closed without merging**